### DISCUSSION TITLE
If I deploy the L1 contract with op-deployer, how do I get the L2OutputOracleProxy address?

### DISCUSSION THREAD

**Discussion by AphobiaCat** [ethereum-optimism/developers#664] [Application Development]
### Did you check the documentation?

- [X] Yes
- [ ] No

### Documentation Feedback (Optional)

How do I get the L2OutputOracleProxy after op-deployer deploys the l1 contract

### What type of issue are you experiencing?

- [ ] Error
- [ ] Feature Request
- [X] General Question

### Issue Description

When I deploy op-proposer I find an option --l2oo-address, which is available in deployment guide manual deployment of l1 contract, but when I deploy l1 contract by op-deployer, I don't find the L2OutputOracleProxy. How can I find it, please?

### Steps to Reproduce

1. Deploy the contract through op-deployer
2. Deploy other components
3. Deploying op-proposer failed because --l2oo-address= could not be found

### Are you using a specific library or SDK?

- [ ] OP-SDK
- [ ] Viem
- [X] Other
- [ ] Not Sure

### Are you using the latest version of the library/SDK?

- [X] Yes
- [ ] No

### Environment Details

version tag v.1.10.0

### What type of support do you need?

- [ ] Technical support for issue resolution
- [ ] Feature implementation guidance
- [ ] General advice or consultation
- [ ] Documentation or learning resources

**Comment by opfocus** [comment]
https://discord.com/channels/667044843901681675/1080862392281481246/1311714401497251871   other developer has answered

**Reply by neodiz** [reply]
> https://discord.com/channels/667044843901681675/1080862392281481246/1311714401497251871 other developer has answered

This link does not work. Can you explain where to get the wallet?

**Reply by opfocus** [reply]
https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-12606204  there is a way to get L1 contracts, but op-deployer maybe not deployed L200

**Reply by LohkoCoder** [reply]
![截屏2025-06-24 17 16 55](https://github.com/user-attachments/assets/1d972b68-b93f-455e-98ac-9a82456cdf16)
In my case, I noticed "L2OutputOracleProxy" address is "0x000..." which probably indicates it't not deployed by op-deployer.
